application/src/main/java/org/opentripplanner/routing/alternativelegs/AlternativeLegs.java

```diff
@@ -39,20 +39,25 @@
  */
 public class AlternativeLegs {
 
+  public enum SearchDirection {
+    NEXT,
+    PREVIOUS,
+  }
+
   public static final int ZERO_COST = 0;
 
   public static List<ScheduledTransitLeg> getAlternativeLegs(
     Leg leg,
     Integer numberLegs,
     TransitService transitService,
-    boolean searchBackward,
+    SearchDirection searchDirection,
```

Comment:
Ideally we should have an enum witch reflect two things:
 - We want the legs **immediately** after or before the given leg.
 - Before/after (next/prev) in *time*. 
 
 So:
 ```
 SearchTime timeLine
 ```
 
 I did not manage to include _immediately after or before_ here. An alternative could be:
 
 `enum Adjacent{ PREVVIOUS, NEXT }` and parameter `relativeInTime : Adjacent`. `Adjacent` could probably be in `org.opentripplanner.framework.time`, but id we later need to change it to something witch is not so generic, then it is better to be local.